packages/one-bx-shell-app/src/app/components/landing/landing.component.html

```diff
@@ -6,23 +6,14 @@
   </fd-layout-panel>
   <fd-layout-panel [columnSpan]="2" fdLayoutGridSpan>
     <fd-layout-panel-body>
-      <h4>Quick Links</h4>
-      <hr/>
-      <a fd-link href="">
-        <fd-icon [size]="'s'" glyph="activity-items"></fd-icon>
-        Request Non-catalog Item</a><br/><br/>
-
-      <a fd-link href="">
-        <fd-icon [size]="'s'" glyph="batch-payments"></fd-icon>
-        Request By Plant</a><br/><br/>
-      <a fd-link href="">
-        <fd-icon [size]="'s'" glyph="cart"></fd-icon>
-        Create Project</a><br/><br/>
+      <fds-iframe-launcher url="http://localhost:5400"
+                            [attrs]="{frameborder:0, width: '100%', height: '210px'}">
+      </fds-iframe-launcher>
     </fd-layout-panel-body>
   </fd-layout-panel>
 
   <fd-layout-panel [columnSpan]="5" fdLayoutGridSpan>
-    <fds-plugin-laucher name="contentItem.recommendedCategories"></fds-plugin-laucher>
+    <app-plugin-laucher name="contentItem.recommendedCategories"></app-plugin-laucher>
```

Comment:
this should be reverted and fds plugin launcher and app plugin launcher component should be removed